index.js

```diff
@@ -51,12 +51,13 @@ module.exports = function metalsmithBabel(options) {
       }
 
       files[filename].contents = SafeBuffer.from(result.code);
-    }
+    });
 
     if (noFilesRenamed) {
-      return;
+      return done();
     }
 
     toFastProperties(files);
+    return done();
```

Comment:
> `done` callback was removed for some reason, making the plugin to stall forever.

No, `done` callback is optional. https://github.com/segmentio/ware/blob/132e961cae2f7d87842fd3a8e0bf540e5ad4f7c9/lib/index.js#L71-L73